resources/assets/ts/common/init.ts

```diff
@@ -12,6 +20,23 @@ $(window).on({
         // isotope.layout();
       }
     });
+    if (!checkInputSupport("datetime-local")) {
```

Comment:
is this for legacy support of input datetime-local elements? If so, I think this should be in a separate branch

Reply:
the datetime-local is not legacy
The datetime is
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local